.travis.yml

```diff
@@ -3,31 +3,28 @@ sudo: false
 language: php
 
 php:
-    - 7.0
-    - 7.1
-    - 7.2
+    - 7.4
+    - 8.0
+    - 8.1
 
 cache:
     directories:
         - $HOME/.composer/cache/files
 
 env:
-    - SYMFONY_VERSION="^3.0"
```

Comment:
❓ Is Symfony 3 really no longer supported?

Reply:
Positive.